docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.1'
+services:
+  docs:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    working_dir: /opt/rf-docs
+    volumes:
+      - ./karma.conf.js:/opt/rf-docs/karma.conf.js
+      - ./protractor.conf.js:/opt/rf-docs/protractor.conf.js
+      - ./.angular-cli.json:/opt/rf-docs/.angular-cli.json
+      - ./package.json:/opt/rf-docs/package.json
+      - ./tsconfig.json:/opt/rf-docs/tsconfig.json
+      - ./tslint.json:/opt/rf-docs/tslint.json
+      - ./yarn.lock:/opt/rf-docs/yarn.lock
+      - ./src:/opt/rf-docs/src
+      - ./prototypes:/opt/rf-docs/prototypes
+      - ./e2e:/opt/rf-docs/e2e
+      - ./dist:/opt/rf-docs/dist
```

Comment:
I think that this is the only volume mount we need. All of the others can go based on the `COPY . /opt/rf-docs` above.